internal/distro/fedora33/distro.go

```diff
@@ -586,14 +588,18 @@ func ostreeCommitAssembler(options distro.ImageOptions, arch distro.Arch) *osbui
 
 // New creates a new distro object, defining the supported architectures and image types
 func New() distro.Distro {
-	return newDistro(defaultName)
+	return newDistro(defaultName, modulePlatformID, ostreeRef)
 }
 
 func NewHostDistro(name string) distro.Distro {
-	return newDistro(name)
+	return newDistro(name, modulePlatformID, ostreeRef)
+}
+
+func NewUnreleasedDistro(name, distroVersion string) distro.Distro {
+	return newDistro(name, "platform:f"+distroVersion, "fedora/"+distroVersion+"/%s/iot")
```

Comment:
I would suggest to rather change the `NewHostDistro()` definition in all distros to accept the `name` and `distroVersion` than an additional Method. This way we could easily decide which Fedora / RHEL distro definition will be used as a fallback in Fedora and RHEL and construct it in the same way whether it is unsupported or not.